### PR TITLE
Clean up rkt mentions in `pkg/kubelet/pleg`

### PR DESCRIPTION
**Pull request by mattjmcnaughton**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Remove the final direct mention of `rkt` in the kubelet codebase.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
```docs
NONE
```

### CODE REVIEW

File: pkg/kubelet/pleg/generic.go [hunk 1]
```diff
@@ -366,7 +366,7 @@ func (g *GenericPLEG) getPodIPs(pid types.UID, status *kubecontainer.PodStatus)
 	}
 
 	if len(status.SandboxStatuses) == 0 {
-		// Without sandboxes (which built-in runtimes like rkt don't report)
+		// If sandboxes are not present, then
```
Comment (mattjmcnaughton) [inline pkg/kubelet/pleg/generic.go]: As a note for reviewers... I'm assuming that we still want to fallback to checking `ContainerStatuses` when no `SandboxStatuses` are present - is that a safe assumption?

Another option would be to remove this `if` statement entirely if we could be confident that `len(status.SandboxStatuses)` would ALWAYS not equal 0 when using the `cri`. I wasn't sure if that assumption was safe.

**Comment by mattjmcnaughton** [comment]
/assign @derekwaynecarr

thanks :) this diff is the last in a series to clean up references to  outdated references to`rkt` in the kubelet.

**Comment by mattjmcnaughton** [comment]
On Sun, Jan 12, 2020 at 09:25:57AM -0800, Kubernetes Prow Robot wrote:
> @mattjmcnaughton: The following tests **failed**, say `/retest` to rerun all failed tests:
> 
> Test name | Commit | Details | Rerun command
> --- | --- | --- | ---
> pull-kubernetes-e2e-kind | c06c25dc6f0970dd31f21225b27e6119e5a11c2b | [link](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87116/pull-kubernetes-e2e-kind/1216396520711000064) | `/test pull-kubernetes-e2e-kind`
> pull-kubernetes-e2e-gce | c06c25dc6f0970dd31f21225b27e6119e5a11c2b | [link](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87116/pull-kubernetes-e2e-gce/1216396520711000065/) | `/test pull-kubernetes-e2e-gce`
> 
> [Full PR test history](https://prow.k8s.io/pr-history?org=kubernetes&repo=kubernetes&pr=87116). [Your PR dashboard](https://gubernator.k8s.io/pr/mattjmcnaughton). Please help us cut down on flakes by [linking to](https://git.k8s.io/community/contributors/devel/sig-testing/flaky-tests.md#filing-issues-for-flaky-tests) an [open issue](https://github.com/kubernetes/kubernetes/issues?q=is:issue+is:open) when you hit one in your PR.
> 
> <details>
> 
> Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
> </details>
> 
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/kubernetes/kubernetes/pull/87116#issuecomment-573437558

/retest

File: pkg/kubelet/pleg/generic.go (hunk 1 above)
Comment (dashpole) [reply pkg/kubelet/pleg/generic.go]: cc @Random-Liu @yujuhong

Comment (yujuhong) [reply pkg/kubelet/pleg/generic.go]: Since rkt is no longer a built-in runtime, and docker(shim) uses CRI, I think it's safe to remove the block entirely.

Comment (mattjmcnaughton) [reply pkg/kubelet/pleg/generic.go]: :+1: will update :)

**Comment by mattjmcnaughton** [comment]
Thanks to some helpful feedback from @yujuhong , the focus of this PR changed. So I'm going to close it and create another one which deletes the rkt specific code block entirely.

**Review by mattjmcnaughton — COMMENTED** [review]
Thanks for the extra info. Since the purpose of this diff is changing, I'm closing this PR and replacing with https://github.com/kubernetes/kubernetes/pull/87193.

**Pull request closed without merging**